<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="leonido-HP-Pavilion-Sleekbook-15-PC" failures="1" tests="3" name="negative_tests" time="3.436" errors="0" timestamp="2021-08-07T17:14:28 IDT">
  <testcase classname="negative_tests.TestEndpoint2" name="request_to_non_existing_id_should_be_rejected" time="3.298"/>
  <testcase classname="negative_tests.TestEndpoint2" name="request_to_non_numeric_id_should_be_rejected" time="3.298"/>
  <testcase classname="negative_tests.TestEndpoint1" name="push_document_with_post_query" time="3.351">
    <failure type="java.lang.AssertionError" message="expected [404] but found [201]">
      <![CDATA[java.lang.AssertionError: expected [404] but found [201]
at negative_tests.NegativeTest.response_is_empty_http_not_found(NegativeTest.java:25)
at negative_tests.TestEndpoint1.push_document_with_post_query(TestEndpoint1.java:27)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:829)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- push_document_with_post_query -->
</testsuite> <!-- negative_tests -->
